develop/scan-images/index.md

```diff
@@ -4,13 +4,17 @@ description: Scan images best practices guide
 keywords: docker scan, scan, images, snyk, vulnerability
 ---
 
-{% include sign-up-cta.html
-  body="You can now get 10 free scans per month as part of your Docker subscription. Sign in to Docker to start scanning your images for vulnerabilities."
-  header-text="This feature requires a Docker subscription"
-  target-url="https://www.docker.com/pricing?utm_source=docker&utm_medium=webreferral&utm_campaign=docs_driven_upgrade_scan"
-%}
-
-This page contains recommendations and best practices for scanning and building secure images.
+> **Log4j 2 CVE-2021-44228**
+>
+> Versions of `docker Scan` earlier than v0.11.0 are not able to detect [Log4j 2
```

Comment:
nit: can we add backquotes around `v0.11.0` ?

Reply:
Fixed